packages/as-transformer/src/transformers/massaExport.ts

```diff
@@ -146,7 +146,7 @@ export class MassaExport {
       wrapper +=
         '  generateEvent(' +
         `\`Result${this.functionName}:` +
-        "'${response.buffer.toString()}'`);\n";
+        "'${byteArrayToBase64(new Uint8Array(response.buffer))}'`);\n";
```

Comment:
Why not directly using response which is an Uint8array?

Reply:
idk